packages/fx-core/src/component/generator/generator.ts

```diff
@@ -90,9 +90,6 @@ export class Generator {
       SolutionName: solutionNameFromVS ?? appName,
       ApiKey: apiKeyActionData,
       OAuth: oauthActionData,
-      enableTestToolByDefault: featureFlagManager.getBooleanValue(FeatureFlags.TestTool)
-        ? "true"
-        : "",
       enableMETestToolByDefault: featureFlagManager.getBooleanValue(FeatureFlags.METestTool)
```

Comment:
Is `METestTool` still in use? If not, what about cleaning in the same PR?

Reply:
Will clean it in following PR since this PR has many changes.